include/cpp-gray/gray.h

```diff
@@ -99,6 +110,13 @@ namespace cppgray
          */
         explicit constexpr operator value_type() const noexcept;
 
+        /**
+         * @brief Conversion to the exact underlying bit representation.
+         *
+         * Assumes 8-bit byte
+         */
+        explicit constexpr operator std::bitset<sizeof(value_type) * 8>() const noexcept;
```

Comment:
Also `std::numeric_limits<value_type>::digits` is better than `sizeof(value_type) * 8`.